odev/commands/database/database.py

```diff
@@ -0,0 +1,111 @@
+from odev.common import args, progress
+from odev.common.commands import GitCommand, LocalDatabaseCommand
+from odev.common.connectors import GitConnector
+from odev.common.logging import logging
+from odev.common.python import PythonEnv
+
+
+logger = logging.getLogger(__name__)
+
+
+class DatabaseSetCommand(LocalDatabaseCommand, GitCommand):
+    """Edit local databases' parameters."""
```

Comment:
```suggestion
    """Edit local databases parameters."""
```
maybe it's just me, but this apostrophe does not look good grammatically to me